app/controllers/admin/visualizations_controller.rb

```
@@ -332,14 +337,14 @@ def show_protected_public_map
     respond_to do |format|
       format.html { render 'public_map', layout: 'application_public_visualization_layout' }
     end
-  rescue
+  rescue => e
```

This is all my fault for copying code when building pass-protected public map pages without reasoning about this catch-all blocks :(
From now on no more "quick copy and paste" of stuff to not waste other's time